lldb/source/Plugins/Process/Windows/Common/ProcessWindows.cpp

```diff
@@ -236,11 +236,18 @@ ProcessWindows::DoAttachToProcessWithID(lldb::pid_t pid,
   return error;
 }
 
-Status ProcessWindows::DoResume() {
+Status ProcessWindows::DoResume(RunDirection direction) {
   Log *log = GetLog(WindowsLog::Process);
   llvm::sys::ScopedLock lock(m_mutex);
   Status error;
 
+  if (direction == RunDirection::eRunReverse) {
```

Comment:
It might be nicer to have a Process::SupportsReverseContinue, then we could check this in PrivateResume before calling DoResume.  That seems more natural than having each plugin return an error in their DoResume.

Reply:
Yes, but ... if we do that, `DoResume()` still needs to report an error if `direction` is not supported, right? And if we have to report an error anyway, then I think `Process::SupportsReverseContinue()` is superfluous.